0.17/files/create-settings.sh

```diff
@@ -232,23 +244,42 @@ serverSettings(){
 }
 
 
-#call the functions to generate and copy the files from the templates.
-if [[ ! -f $CONFIG/server-settings.json ]]; then
-	# Copy default settings if server-settings.json doesn't exist
-	serverSettings
-	cp ./server-settings-template.json "$CONFIG/server-settings.json"
-fi
 
-if [[ ! -f $CONFIG/map-gen-settings.json ]]; then
+if [[ $FORCE_GENERATE_SETTINGS_FILES ]]
+then
+	#generate new files, regardless of if a file already exists.
+	serverSettings
+	mapSettings
 	mapGenSettings
+	rm -f "$CONFIG/server-settings.json"
```

Comment:
I don't think it is a good idea to just delete the old configs.

Reply:
Files that exist will now get moved to .json.YYYY.MM.DD.HH.MM.SS

Reply:
I think this is a good solution to not loose any old settings. :+1: